totp/__init__.py

```diff
@@ -110,21 +130,34 @@ def generate_token(path, seconds=0):
     # Remove the trailing newline or any other custom data users might have
     # saved:
     pass_entry = pass_entry.splitlines()
-    secret = pass_entry[0]
+
+    secret = return_secret(pass_entry[0])
 
     digits = get_length(pass_entry)
     token = onetimepass.get_totp(secret, as_string=True, token_length=digits,
                                  clock=clock)
 
-    print(token.decode())
+    print("The totp token for " + sys.argv[1] + " is:\n" + token.decode())
```

Comment:
This output is problematic because it breaks scripts that rely on totp's output and alike.
Simply outputting the code is best, the wording is kinda redundant.

Reply:
Not a problem. I imagined it might be. Ill remove it.